recipes/ipychart/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "ipychart" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ac01fbca69039e5631d1f2579edca80f47f15da3132df9aa7259cafd9e7e2846
+
+build:
+  number: 0
```

Comment:
this looks like it can probably be `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python